test/common.bash

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+dump_stderr_console() {
+  printf '## %s\n' "${stderr_lines[@]}" >&3
```

Comment:
Disclaimer: I've never used 'bats' before.
My question here is about using file descriptor "3". Essentially, how does this work? Is this related to bats?

Reply:
Perfect! Thanks for the explanation.